fixtures/fixture.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
Waarom zit deze fixture er in? Waar is het voor?

Reply:
Die zal nodig zijn voor travis, maar voor de Docker containers op zichzelf nog niet, dus ik zal hem hier uit halen